vaadin-time-picker-flow-parent/vaadin-time-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/timepicker/tests/TimePickerValidationBinderIT.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2000-2022 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.timepicker.tests;
+
+import com.vaadin.flow.component.timepicker.testbench.TimePickerElement;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.tests.AbstractComponentIT;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+
+import static com.vaadin.flow.component.timepicker.tests.TimePickerValidationBinderPage.ATTACH_BINDER_BUTTON;
+import static com.vaadin.flow.component.timepicker.tests.TimePickerValidationBinderPage.MAX_VALUE_BUTTON;
+import static com.vaadin.flow.component.timepicker.tests.TimePickerValidationBinderPage.MIN_VALUE_BUTTON;
+import static com.vaadin.flow.component.timepicker.tests.TimePickerValidationBinderPage.REQUIRED_ERROR_MESSAGE;
+import static com.vaadin.flow.component.timepicker.tests.TimePickerValidationBinderPage.SERVER_VALIDITY_STATE;
+import static com.vaadin.flow.component.timepicker.tests.TimePickerValidationBinderPage.SERVER_VALIDITY_STATE_BUTTON;
+import static com.vaadin.flow.component.timepicker.tests.TimePickerValidationBinderPage.UNEXPECTED_VALUE_ERROR_MESSAGE;
+
+/**
+ * Integration tests for validation with binder.
+ */
+@TestPath("vaadin-time-picker/validation-binder")
+public class TimePickerValidationBinderIT extends AbstractComponentIT {
+
+    private TimePickerElement field;
+
+    @Before
+    public void init() {
+        open();
+        field = $(TimePickerElement.class).waitForFirst();
+        findElement(By.id(ATTACH_BINDER_BUTTON)).click();
+    }
+
+    @Test
+    public void required_fieldIsInitiallyValid() {
+        assertClientValid(true);
+        assertServerValid(true);
+        assertErrorMessage(null);
+    }
+
+    @Test
+    public void required_triggerInputBlur_assertValidity() {
+        field.sendKeys(Keys.TAB);
+        assertServerValid(false);
+        assertClientValid(false);
+        assertErrorMessage(REQUIRED_ERROR_MESSAGE);
+    }
```

Comment:
This test is currently failing. The `required` constraint fails on the client and it makes the constraint validation fail on the server instead of the `Binder` constraints.